src/app/components/text-image-component/text-image-component.component.html

```diff
@@ -0,0 +1,6 @@
+<div [ngClass] = "{'container': true, 'right': position, 'left': !position}">
```

Comment:
If you want to apply the "container" class to all components of type text-image you can just write as usual class="container" then put the [ngClass]